hw/constraints_amdc_revd.xdc

```diff
@@ -9,319 +9,324 @@
 # RGB LED #
 ###########
 
-set_property IOSTANDARD LVCMOS18 [get_ports user_led_din];
+set_property IOSTANDARD LVCMOS18 [get_ports {user_led_din[0]}]
 
-set_property PACKAGE_PIN R8 [get_ports user_led_din]; # JX2-14
+set_property PACKAGE_PIN R8 [get_ports {user_led_din[0]}]
 
 
 ###########
 # ENCODER #
 ###########
 
-set_property IOSTANDARD LVCMOS18 [get_ports encoder_1a];
-set_property IOSTANDARD LVCMOS18 [get_ports encoder_1b];
-set_property IOSTANDARD LVCMOS18 [get_ports encoder_1z];
-set_property IOSTANDARD LVCMOS18 [get_ports encoder_2a];
-set_property IOSTANDARD LVCMOS18 [get_ports encoder_2b];
-set_property IOSTANDARD LVCMOS18 [get_ports encoder_2z];
+set_property IOSTANDARD LVCMOS18 [get_ports encoder_1a]
+set_property IOSTANDARD LVCMOS18 [get_ports encoder_1b]
+set_property IOSTANDARD LVCMOS18 [get_ports encoder_1z]
+set_property IOSTANDARD LVCMOS18 [get_ports encoder_2a]
+set_property IOSTANDARD LVCMOS18 [get_ports encoder_2b]
+set_property IOSTANDARD LVCMOS18 [get_ports encoder_2z]
 
-set_property PACKAGE_PIN A2  [get_ports encoder_1a]; # JX1-54
-set_property PACKAGE_PIN A1  [get_ports encoder_1b]; # JX1-56
-set_property PACKAGE_PIN D7  [get_ports encoder_1z]; # JX1-62
-set_property PACKAGE_PIN D6  [get_ports encoder_2a]; # JX1-64
-set_property PACKAGE_PIN A5  [get_ports encoder_2b]; # JX1-68
-set_property PACKAGE_PIN A4  [get_ports encoder_2z]; # JX1-70
+set_property PACKAGE_PIN A2 [get_ports encoder_1a]
+set_property PACKAGE_PIN A1 [get_ports encoder_1b]
+set_property PACKAGE_PIN D7 [get_ports encoder_1z]
+set_property PACKAGE_PIN D6 [get_ports encoder_2a]
+set_property PACKAGE_PIN A5 [get_ports encoder_2b]
+set_property PACKAGE_PIN A4 [get_ports encoder_2z]
```

Comment:
The changes in this file seem to only remove the comments for which pin of the Picozed the corresponding port is connected to. I think this file can be reverted so the reference doesn't get removed. Everything else looks good to me.

Reply:
Reverted the file